pkgs/development/haskell-modules/configuration-ghc-7.10.x.nix

```diff
@@ -36,6 +36,37 @@ self: super: {
   # Cabal_1_22_1_1 requires filepath >=1 && <1.4
   cabal-install = dontCheck (super.cabal-install.override { Cabal = null; });
 
+  HStringTemplate = self.HStringTemplate_0_8_3;
+
+  # update gtk2hs versions, until hackage-pacakges.nix is updated.
+  cairo = overrideCabal super.cairo (drv: {
```

Comment:
Please don't hide package updates in the configuration files! This is bound to create a lot of confusion when `hackage-packages.nix` is updated, because the changes in there will be meaningless.